src/control/server/mgmt_system_test.go

```diff
@@ -183,7 +187,22 @@ func TestServer_MgmtSvc_ClusterEvent(t *testing.T) {
 				}
 			}
 
-			gotResp, gotErr := mgmtSvc.ClusterEvent(context.TODO(), tc.req)
+			mockLookupAddr := func(addr string) ([]string, error) {
+				return map[string][]string{
+					"192.168.0.1": {"foo.bar.com"},
+				}[addr], nil
+			}
+
+			var pbReq *mgmtpb.ClusterEventReq
+			if tc.notifyReq != nil {
+				var err error
+				pbReq, err = tc.notifyReq.ToClusterEventReq(mockLookupAddr)
+				if err != nil {
+					t.Fatal(err)
+				}
+			}
```

Comment:
Would be better to change the tc.notifyReq to use the ClusterEventReq directly, rather than doing an extra conversion here.